src/main/java/lotto/service/WinningAnalyzer.java

```diff
@@ -1,26 +1,27 @@
 package lotto.service;
 
-import lotto.model.Lotto;
+import lotto.model.BoughtLotto;
 import lotto.model.WinningInfo;
+import lotto.model.WinningLotto;
 
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-public class LottoAnalyzer {
-    private final static int[] prize = {0, 2_000_000_000, 30_000_000, 1_500_000, 50_000, 5_000};
+public class WinningAnalyzer {
+    private static final int[] prize = {0, 2_000_000_000, 30_000_000, 1_500_000, 50_000, 5_000};
 
     private final Map<Integer, WinningInfo> winningStatics;
```

Comment:
통계와 관련된 변수이니, 이름을 `-statistics`로 변경해보면 좋을 것 같습니다.
```suggestion
    private final Map<Integer, WinningInfo> winningStatistics;
```